### ISSUE TITLE
As institute manager, I want to see up-to-date user status

### ISSUE THREAD

**Issue by goomens** [UvA-FNWI/DevicePortal#41]
Sync user state with DataNose so that
- New users are added so that devices can be assigned to them
- Users who have left the UvA get an inactive status

**Comment by muzzi11** [comment]
Add post user endpoint

**Comment by muzzi11** [comment]
Added a new config variable:
"Api": {
    "Key": ""
}

This key must be provided inside the "X-Api-Key" header.

To allow outside access CORS settings have been added, these are very permissive at the moment for testing purposes. It's probably best to restrict these to DataNose only.

For testing purposes I've created a stand alone Swagger API documentation file, this could be used as a base for further API expansion. I see no way of attaching a file to this post so I will send this file over Teams (maybe add it to the repo at some point).
